### PR TITLE
Raf indexer accounts application

### PR DESCRIPTION
**Pull request by rfustino**
Samples and doc updated for accounts search for application. Paging example as well.

### CODE REVIEW

File: examples/indexer/java/AccountsAppID.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import com.algorand.algosdk.v2.client.common.Client;
+
+import org.json.JSONObject;
+
+public class AccountsAppID {
+    public Client indexerInstance = null;
+    // utility function to connect to a node
+    private Client connectToNetwork() {
+        
+        // TODO  remove
```
Comment (ryanRfox) [inline examples/indexer/java/AccountsAppID.java:18]: Remove

File: examples/indexer/java/AccountsAppID.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    private Client connectToNetwork() {
+        
+        // TODO  remove    
+        // final String INDEXER_API_ADDR = "https://indexer-internal-betanet.aws.algodev.network/";
+        // final String INDEXER_TOKEN = "YddOUGbAjHLr1uPZtZwHOvMDmXvR1Zvw1f3Roj2PT1ufenXbNyIxIz0IeznrLbDsF";
+        // final int INDEXER_API_PORT = 443;
+
+        final String INDEXER_API_ADDR = "http://localhost";
+        final String INDEXER_TOKEN = "";
+        final int INDEXER_API_PORT = 59998;
```
Comment (ryanRfox) [inline examples/indexer/java/AccountsAppID.java:25]: Should be 8980 throughout 

Reference default setting:
https://github.com/algorand/indexer/blob/7e7f1c34d6f930a424a6030750fe1d7a54006d2d/cmd/algorand-indexer/daemon.go#L115

File: examples/indexer/javascript/accountsApplicationID.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+// verify installed version
+// npm list algosdk
+
+const algosdk = require('algosdk');
+const chalk = require('chalk');
+// const indexer_token = "";
+// const indexer_server = "http://localhost";
+// const indexer_port = 8980;
+
+// const indexer_server = "https://testnet-algorand.api.purestake.io/idx2/";
```
Comment (ryanRfox) [inline examples/indexer/javascript/accountsApplicationID.js:12]: Remove

**Review by ryanRfox — CHANGES_REQUESTED** [review]
[ ] Set port to 8980 throughout
[ ] Remove references to Pure Stake API throughout
[ ] Remove references to AWS DB throughout
[ ] Remove extraneous commented lines with testing parameters

**Comment by ejbaran** [comment]
PR is out of date. Closing.

**Pull request closed without merging**